### PR TITLE
Refactor rename endpoint to update

### LINKED ISSUE #45: Change Rename wishlist endpoint to Update Wishlist.

### PR DESCRIPTION
**Pull request by gabeorlanski**
Fixed #45

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -273,7 +273,7 @@ def update_product(wishlist_id, item_id):
 
     new_name = body.get("product_name", None)
     if not new_name:
-        abort(status.HTTP_400_BAD_REQUEST, "No product name passed to rename.")
+        abort(status.HTTP_400_BAD_REQUEST, "No product name passed to update.")
```
Comment (jaikarnyu) [inline service/routes.py:276]: Please change the name of the method to update_item.

File: service/routes.py [hunk 2]
```diff
@@ -273,7 +273,7 @@ def update_product(wishlist_id, item_id):
...
```
Comment (jaikarnyu) [inline service/routes.py:277]: Please add code for updating other fields like rank, price , quantity along with name.

File: service/routes.py [hunk 3]
```diff
@@ -273,7 +273,7 @@ def update_product(wishlist_id, item_id):
...
 
     wishlist_product.name = new_name
     wishlist_product.update()
```
Comment (jaikarnyu) [inline service/routes.py:279]: Please check if changes to testcases are needed for the above changes.

File: service/routes.py (hunk 2 above)
Comment (gabeorlanski) [reply service/routes.py:277]: That is not what this issue is for, and thus should be reflected in a new issue.

**Review by jaikarnyu — APPROVED** [review]
looks good !

**Pull request merged**